### PR TITLE
Re-enabling Node 8 which is in LTS until December 2019.

### PR DESCRIPTION
**Pull request by bigkraig**

### CODE REVIEW

**Comment by bigkraig** [comment]
@sindresorhus I've re-enabled Node 8 in this branch since its still in an LTS state through the rest of the year. Mostly to support https://github.com/danger/danger-js/pull/855. I hope this isn't a problem. Thanks!

File: package.json [hunk 1]
```diff
@@ -10,7 +10,7 @@
 		"url": "sindresorhus.com"
 	},
 	"engines": {
-		"node": ">=10"
+		"node": ">=8.9.0"
```
[the remark this answers is not in the document]
Comment (bigkraig) [reply package.json]: Could go with 8 but the danger-js repo I am trying to get a PR into had tests for 8.4, 8.12, and 8.

**Comment by bigkraig** [comment]
@sholladay 6 is EOL in a few weeks but 8 at least runs until December. TBH I feel the same way but am just trying to get some updates to another repo and they prefer to support LTS.

I've updated the repo to just require "8" now.

image

**Comment by sindresorhus** [comment]
Fixed in https://github.com/sindresorhus/ky/releases/tag/v0.10.0

There's also a new version of `ky-universal` that re-enables Node.js 8 support (no polyfilling needed): https://github.com/sindresorhus/ky-universal/releases/tag/v0.2.0

**Pull request closed without merging**